packages/features/form-builder/Components.tsx

```diff
@@ -272,7 +272,7 @@ export const Components: Record<BookingFieldType, Component> = {
                 label={option.label}
                 key={`option.${i}.radio`}
                 value={option.label}
-                id={`option.${i}.radio`}
+                id={`${name}.option.${i}.radio`}
```

Comment:
It won't be required if we move radio input inside label in https://github.com/calcom/cal.com/blob/2752061263fafa93cbaa756ec1f77e2dc5b8534b/packages/ui/form/radio-area/Radio.tsx#L56

@sean-brydon What do you think about it?